source/Nuke.Common/Tools/OctoVersion/OctoVersionAttribute.cs

```diff
@@ -22,27 +24,232 @@ namespace Nuke.Common.Tools.OctoVersion
     [UsedImplicitly(ImplicitUseKindFlags.Default)]
     public class OctoVersionAttribute : ValueInjectionAttributeBase
     {
+        /// <summary>
```

Comment:
Member layout should be first all fields, and then the attributes

Reply:
Done